app/components/UserValidator/index.tsx

```diff
@@ -191,7 +191,7 @@ const Validator = ({ handleSelect, validateAbakusGroup }: Props) => {
           setScanResults([]);
         }}
       >
-        <h1>Scan ABA-ID</h1>
```

Comment:
ye, perhaps no

Reply:
Heh, I have now gone through all instances of `EntityId` and fixed all the wrong replacements:)